qiime2/sdk/usage.py

```diff
@@ -385,6 +805,14 @@ def _assert_has_line_matching_(self, ref, label, path, expression):
 
 
 class ExecutionUsage(Usage):
+    """
+    Execute and test rendered examples.
+
+    See Also
+    --------
+    qiime2.plugin.tests.TestUsage : Unit tests using this driver.
```

Comment:
I don't see a suite by this name at this path - can you either update the name here, or fix the name in the tests?